src/hub_predtimechart/hub_config_ptc.py

```diff
@@ -90,8 +101,8 @@ def model_output_file_for_ref_date(self, model_id: str, reference_date: str) ->
         Returns a Path to the model output file corresponding to `model_id` and `reference_date`. Returns None if none
         found.
         """
-        poss_output_files = [self.hub_dir / 'model-output' / model_id / f"{reference_date}-{model_id}.csv",
-                             self.hub_dir / 'model-output' / model_id / f"{reference_date}-{model_id}.parquet"]
+        poss_output_files = [self.hub_path / 'model-output' / model_id / f"{reference_date}-{model_id}.csv",
+                             self.hub_path / 'model-output' / model_id / f"{reference_date}-{model_id}.parquet"]
```

Comment:
I know this is not related to this PR but this seems to limit dashboards to hubs where `round_id_from_variable` is true and that variable is a date (e.g. `reference_date`). This seems sensible but is not listed as a limitation in https://github.com/hubverse-org/hub-dashboard-predtimechart/blob/311f8fcde17e4f2b2e2307e70859a4fa2f1951c7/README.md?plain=1#L42-L55 or in https://docs.hubverse.io/en/latest/user-guide/dashboards.html (it's sort of implied in a round about way). Would be a good idea to mention that explicitly. Perhaps adding a check for it to ensure there are no parsing errors of file names that may not be using names for round IDs instead of a reference date (rare but technically allowed) might also be sensible.